src/feedbinurlreader.cpp

```diff
@@ -0,0 +1,58 @@
+#include "feedbinurlreader.h"
+
+#include "fileurlreader.h"
+#include "logger.h"
+#include "remoteapi.h"
+#include "utils.h"
+
+namespace newsboat {
+
+FeedbinUrlReader::FeedbinUrlReader(const std::string& url_file, RemoteApi* a)
+	: file(url_file)
+	, api(a)
+{
+}
+
+nonstd::optional<utils::ReadTextFileError> FeedbinUrlReader::reload()
+{
+	urls.clear();
+	tags.clear();
+	alltags.clear();
+
+	FileUrlReader ur(file);
+	const auto error_message = ur.reload();
+	if (error_message.has_value()) {
+		LOG(Level::DEBUG, "Reloading failed: %s", error_message.value().message);
+		// Ignore errors for now: https://github.com/newsboat/newsboat/issues/1273
+	}
+
+	const std::vector<std::string>& file_urls(ur.get_urls());
+	LOG(Level::INFO, "URL count: %s", file_urls.size());
```

Comment:
I just noticed this when compiling. It's giving me an error, could be some system specific thing (I'm using mac) since the CI passed.

<details>
<summary>Error message snippet</summary>

```
include/logger.h:19:19: error: call to 'fmt' is ambiguous
                log_internal(l, strprintf::fmt(format, args...));
                                ^~~~~~~~~~~~~~
src/feedbinurlreader.cpp:30:2: note: in instantiation of function template specialization 'newsboat::logger::log<unsigned long>' requested here
        LOG(Level::INFO, "URL count: %s", file_urls.size());
```

</details>

But this drew my attention to the formatting specifier: It seems incorrect, `%s` is for C-string but the argument is a number.

Reply:
Right, I think we had this problem with macOS before. Apparently `size_t` is no aliased to one of the `uint*_t` types, but that's just a guess which I never found time to confirm. So we employ a kludge:

```c++
LOG(Level::INFO, "URL count: " PRIu64, static_cast<uint64_t>(file_urls.size()));
```

Since we don't have macOS in CI anymore (#2280), can you please double-check that this fix works? Thank you!

Reply:
Yes, that compiles just fine after adding the include `#include <cinttypes>`.

The `PRIu64` doesn't include `%`, so that is needed for runtime correctness:
```cpp
LOG(Level::INFO, "URL count: %" PRIu64, static_cast<uint64_t>(file_urls.size()));
```

Ps. looks like on mac `uint64_t` is `unsigned long long`, where as `size_t` is `unsigned long` (both are same size).